### PR TITLE
Show Feature details in Sync Status UI

### PR DESCRIPTION
**Pull request by scolsen**
Screen Shot 2021-07-09 at 4 07 57 PM

### CODE REVIEW

**Comment by scolsen** [comment]
@gino-m ready for a review! Thanks for your help!

**Comment by gino-m** [comment]
@scolsen From the screenshot look like it's still dumping the feature id instead of the caption. Could you PTAL?

**Review by gino-m — CHANGES_REQUESTED** [review]

File: gnd/src/main/java/com/google/android/gnd/ui/syncstatus/SyncStatusListAdapter.java [hunk 1]
```diff
@@ -62,6 +72,12 @@ public void onBindViewHolder(SyncStatusViewHolder viewHolder, int position) {
             .append(dateFormat.format(mutation.getClientTimestamp()))
             .append(' ')
             .append(timeFormat.format(mutation.getClientTimestamp()))
+            .append("\n")
+            .append(featureHelper.getLabel(Optional.of(feature)))
+            .append(' ')
+            .append(feature.getId())
```
[the remark this answers is not in the document]
Comment (scolsen) [reply gnd/src/main/java/com/google/android/gnd/ui/syncstatus/SyncStatusListAdapter.java:78]: This method doesn't seem to exist.

[the remark this answers is not in the document]
Comment (scolsen) [reply gnd/src/main/java/com/google/android/gnd/ui/syncstatus/SyncStatusListAdapter.java:78]: @gino-m do you have it defined in some branch other than master?

[the remark this answers is not in the document]
Comment (gino-m) [reply gnd/src/main/java/com/google/android/gnd/ui/syncstatus/SyncStatusListAdapter.java:78]: Errata corrige: `featureHelper.getCaption(...)`

[the remark this answers is not in the document]
Comment (scolsen) [reply gnd/src/main/java/com/google/android/gnd/ui/syncstatus/SyncStatusListAdapter.java:78]: Oh got it, thanks!

[the remark this answers is not in the document]
Comment (scolsen) [reply gnd/src/main/java/com/google/android/gnd/ui/syncstatus/SyncStatusListAdapter.java:78]: Looks like `getCaption` is private and `getLabel` calls `getCaption` anyway.

[the remark this answers is not in the document]
Comment (gino-m) [reply gnd/src/main/java/com/google/android/gnd/ui/syncstatus/SyncStatusListAdapter.java:78]: Please lmk when it's ready for another look!

**Comment by scolsen** [comment]
@gino-m ready for another look

**Review by gino-m — APPROVED** [review]

**Pull request merged**